examples/poc/basic_deployment/main.tf

```diff
@@ -67,6 +68,7 @@ module "hub" {
   version                             = "1.3.6" # latest release tag
   friendly_name                       = join("-", [local.deployment_name_salted, "hub", "primary"])
   subnet_id                           = module.vpc.public_subnets[0]
+  security_group_id                   = local.security_group_id
```

Comment:
security_group_id should be an optional argument. It should be set to null by default